src/transformers/commands/chat.py

```diff
@@ -276,93 +156,54 @@ def stream_output(self, output_stream):
         self._console.print()
         return text
 
-    def input(self):
+    def input(self) -> str:
+        """Gets user input from the console."""
         input = self._console.input(f"[bold red]<{self.user_name}>:\n")
         self._console.print()
         return input
 
     def clear(self):
+        """Clears the console."""
         self._console.clear()
 
-    def print_user_message(self, text):
+    def print_user_message(self, text: str):
+        """Prints a user message to the console."""
         self._console.print(f"[bold red]<{self.user_name}>:[/ bold red]\n{text}")
         self._console.print()
 
-    def print_green(self, text):
-        self._console.print(f"[bold green]{text}")
+    def print_color(self, text: str, color: str):
+        """Prints text in a given color to the console."""
+        self._console.print(f"[bold {color}]{text}")
         self._console.print()
 
-    def print_red(self, text):
-        self._console.print(f"[bold red]{text}")
-        self._console.print()
-
-    def print_help(self):
-        self._console.print(Markdown(HELP_STRING))
+    def print_help(self, minimal: bool = False):
+        """Prints the help message to the console."""
+        self._console.print(Markdown(HELP_STRING_MINIMAL if minimal else HELP_STRING))
         self._console.print()
 
 
 @dataclass
 class ChatArguments:
     r"""
-    Arguments for the chat script.
-
-    Args:
-        model_name_or_path (`str`):
-            Name of the pre-trained model.
-        user (`str` or `None`, *optional*, defaults to `None`):
-            Username to display in chat interface.
-        system_prompt (`str` or `None`, *optional*, defaults to `None`):
-            System prompt.
```

Comment:
Replacing this entire block with the class attribute help strings is extremely satisfying, assuming it still prints well for users!

Reply:
Yup! There are too many args, but that's a problem for a follow-up PR